cmsimple/seofuncs.php

```diff
@@ -34,8 +34,15 @@ function XH_URI_Cleaning()
     $parts = parse_url(CMSIMPLE_URL);
     $scheme = $parts['scheme'];
     $host = $parts['host'];
+    $port = '';
+    if (!empty($parts['port'])) {
+        $port = ':' . $parts['port'];
+    }
     $path = $parts['path'];
-    $query_str = $_SERVER['QUERY_STRING'];
+    $query_str = '';
+    if (isset($_SERVER['QUERY_STRING'])) {
+        $query_str = $_SERVER['QUERY_STRING'];
+    }
```

Comment:
Von PHP 7.0.0 an könnte man vereinfachen:
```suggestion
    $query_str = $_SERVER['QUERY_STRING'] ?? '';
```

Reply:
Ok, aber das wird uns für die 1.7.4 noch nichts nutzen.
Für die 1.8 gern.
Haben wir denn für die 1.8 schon einen festen Plan, was die Systemvoraussetzungen betrifft?

Reply:
Nein; das sollten wir gut überlegen; es gibt, glaube ich, schon noch einige die PHP 5(.6) nutzen, obwohl ich sehr gerne wenigstens PHP 7.0 fordern würde. Aber diese Diskussion sollten wir auch wegen der Sichtbarkeit wohl besser im Forum führen.